### ISSUE TITLE
Command "/root/.local/share/letsencrypt/bin/python2.7 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-wKPWZS/cffi/setup.py';exec(compile(getattr(tokenize, 'open', ope

### ISSUE THREAD

**Issue by strugglingyouth** [certbot/certbot#2857]
![qq 20160417102755](https://cloud.githubusercontent.com/assets/10249666/14584738/35de5de8-0487-11e6-8bdf-b0a46da74bcb.png)

**Comment by strugglingyouth** [comment]
./letsencrypt-auto certonly --standalone --email 630441839@qq.com -d 17linux8.com


run this command then  。。。

**Comment by robertsky** [comment]
same issue? #2804

**Comment by sorcer1122** [comment]
I am having exactly the same issue and nothing helps. Is there any fix?

**Comment by robertsky** [comment]
i managed to get it running... by adding a swapfile to make the memory allocation up to at least 2GB. not sure if it is a fix that works for sure, but it does.

**Comment by sorcer1122** [comment]
Unfortunately, I am not able to add any swaps on my VPS. Is there another solution?

**Comment by bmw** [comment]
Your version of `pip` is unlikely to be the source of your problems. For security/stability reasons, `letsencrypt-auto`, installs specific versions of Python packages that we have tested. With that said, if you want to upgrade your `pip` version, run:

```
${XDG_DATA_HOME:-~/.local/share}/letsencrypt/bin/pip install -U pip
```

**Comment by SwartzCr** [comment]
Could the people who commented upthread let us know if they're still having issues? @sorcer1122 @robertsky @strugglingyouth

**Comment by bmw** [comment]
This should be fixed in our next release with #4978.
